manager-bundle/src/Console/ContaoApplication.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\Console;
+
+use Contao\CoreBundle\Util\PackageUtil;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class ContaoApplication extends Application
+{
+    public function __construct(KernelInterface $kernel)
+    {
+        parent::__construct($kernel);
+
+        $this->setName('Contao Managed Edition');
+        $this->setVersion(PackageUtil::getContaoVersion());
+
+        $inputDefinition = $this->getDefinition();
+        $options = $inputDefinition->getOptions();
+
+        foreach ($options as $k => $option) {
+            if ('no-debug' === $option->getName()) {
+                // We do not support the no-debug option, so unset it
+                unset($options[$k]);
+                break;
+            }
+        }
+
+        $inputDefinition->setOptions($options);
+    }
+
+    public static function createConsoleInput()
+    {
+        $argv = $_SERVER['argv'];
+
+        // Ignore the --no-debug option
+        foreach ($argv as $k => $v) {
+            if ($v === '--no-debug') {
+                unset($argv[$k]);
```

Comment:
This means that if someone passes the `--no-debug` option, they will not be notified that the option does not exist, right? Not sure if this is good.

Reply:
That's correct. Do you think we should show a console warning?

Reply:
No, I think we should remove the `createConsoleInput()` method again, so the regular exception is thrown. I realize that this means that we cannot use the Symfony Deployer receipt without adjustment anymore, but after our discussion a couple of days ago, I came to the conclusion that a Contao Deployer receipt would solve this way better. 🙄

Reply:
Well we can do that, but not the fromInput method but the one where the input is created. I‘ll adjust it tomorrow

Reply:
The `fromInput()` method does not need to be adjusted. We just have to replace `$input = ContaoApplication::createConsoleInput();` with `$input = new ArgvInput();` again in the `contao-console` file.